services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx

```diff
@@ -61,6 +67,13 @@ export const EnvironmentCard: React.FC<{ environment: Environment }> = (props) =
                         icon={<Locks />}
                         onClick={addLock}
                     />
+                    <div>
+                        <Button
+                            className="environment-action"
+                            label={'Display version for ' + environment.name}
```

Comment:
```suggestion
                            label={'Display versions for ' + environment.name}
```
Should be more than one, right?